src/pages/App.tsx

```diff
@@ -87,10 +87,10 @@ const BodyWrapper = styled.div<{ isAboutPage?: boolean }>`
   overflow-y: auto;
   overflow-x: hidden;
 `
-
+const AppPaths = { SWAP_LEGACY: '/swap-legacy', ABOUT: '/about', SWAP: '/swap' }
```

Comment:
Why only 3 path here?

Reply:
it prevents hardcode.  when we have more time, we can put all paths into it.